### PR TITLE
fix ldif syntax and add idnsTemplateAttribute

### PR DESCRIPTION
**Pull request by stutiredboy**
schema.ldif lost some white space in the line end.
schema.ldif lost the idnsTemplateAttribute definitition

### CODE REVIEW

File: doc/schema.ldif [hunk 1]
```diff
@@ -362,9 +362,16 @@ attributeTypes: ( 2.16.840.1.113730.3.8.5.31
  NAME 'idnsServerId'
  DESC 'DNS server identifier'
  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
- EQUALITY caseIgnoreMatch
+ EQUALITY caseIgnoreMatch
```
Comment (MartinBasti) [inline doc/schema.ldif:365]: Please don't add whitespace at the end of line

File: doc/schema.ldif [hunk 2]
```diff
@@ -362,9 +362,16 @@ attributeTypes: ( 2.16.840.1.113730.3.8.5.31
  NAME 'idnsServerId'
  DESC 'DNS server identifier'
  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
- EQUALITY caseIgnoreMatch
+ EQUALITY caseIgnoreMatch 
  SINGLE-VALUE )
 #
+olcattributeTypes: ( 2.16.840.1.113730.3.8.5.29
```
Comment (MartinBasti) [inline doc/schema.ldif]: You have typo there, 'olcattribute''

File: doc/schema.ldif [hunk 3]
```diff
@@ -426,6 +433,6 @@ objectClasses: ( 2.16.840.1.113730.3.8.6.6
 objectClasses: ( 2.16.840.1.113730.3.8.6.5 
  NAME 'idnsTemplateObject' 
  DESC 'Template object for dynamic DNS attribute generation' 
- SUP top
+ SUP top
```
Comment (MartinBasti) [inline doc/schema.ldif:436]: Same here, no extra whitespace please

File: doc/schema.ldif [hunk 4]
```diff
@@ -365,7 +365,7 @@ attributeTypes: ( 2.16.840.1.113730.3.8.5.31
  EQUALITY caseIgnoreMatch 
  SINGLE-VALUE )
 #
-olcattributeTypes: ( 2.16.840.1.113730.3.8.5.29 
+attributeTypes: ( 2.16.840.1.113730.3.8.5.29
```
Comment (MartinBasti) [inline doc/schema.ldif:368]: Please merge this change to previous patch

**Review by MartinBasti — CHANGES_REQUESTED** [review]

**Comment by MartinBasti** [comment]
Hello, I wrote inline comments. Please set proper author name in commit (no root please)

**Comment by pspacek** [comment]
@mbasti-rh , the white-space at the end of line is required here because the first space at the beginning of line will be consumed by LDIF parser. I agree that root should not be author of the commit :-)

**Comment by stutiredboy** [comment]
so sorry for the root committer, but the white space is needed by OpenLDAP LDIF parser. I have fixed the ugly pull request. :-)

Thanks.

**Review by pspacek — APPROVED** [review]
ACK, thank you!

**Comment by pspacek** [comment]
Thanks! I've commited the fix as 17711141882aca3847a5daba2292bcbcc471ec63.

**Pull request closed without merging**